assets/scss/styles.scss

```diff
@@ -21,29 +23,53 @@ $padding: 15px;
   @include grid-column( 2/3 );
   padding: $padding;
 }
+.column-three-quarters {
+  @include grid-column( 3/4 );
+  padding: $padding;
+}
 
 /* Header overrides */
 @media (min-width: 480px) {
-  #global-header.with-proposition .header-wrapper .header-global {
-    float: left;
-    width: 10%;
-  }
-  #global-header.with-proposition .header-wrapper .header-proposition {
-    width: 90%;
-    float: left;
-  }
-  #global-header .header-proposition {
-    padding-top: 0;
-  }
-  #global-header .header-proposition #proposition-name {
-    font-family: "nta", Arial, sans-serif;
-    font-size: 36px;
-    line-height: 1.25;
-    font-weight: 400;
-    text-transform: none;
-    font-weight: bold;
-    color: white;
-    text-decoration: none;
+  #global-header {
+    &.with-proposition {
+      .header-wrapper {
+        .header-global {
+          float: left;
+          width: 10%;
+        }
+        .header-proposition {
+          width: 90%;
+          float: left;
+        }
+      }
+    }
+    .header-proposition {
+      padding-top: 0;
+      #proposition-name {
+        font-family: "nta", Arial, sans-serif;
+        font-size: 36px;
```

Comment:
Please don't use font sizes in px inherit from a header if thats what you're trying to do?